### PR TITLE
Added dutch translations

### PR DESCRIPTION
**Pull request by ikulis**
Added dutch translations.

### CODE REVIEW

File: src/translations/validators.nl.xlf [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                <source>This value is not a valid BBAN.</source>
+                <target>Deze waarde is geen geldige BBAN.</target>
+            </trans-unit>
+            <trans-unit id="cif_message">
+                <source>This value is not a valid CIF.</source>
+                <target>Deze waarde is geen geldige CIF.</target>
+            </trans-unit>
+            <trans-unit id="credit_card_message">
+                <source>This value is not a valid credit card scheme.</source>
+                <target>Deze waarde is geen geldig credit card nummer .</target>
```
Comment (sstok) [inline src/translations/validators.nl.xlf:15]:
```suggestion
                <target>Deze waarde is geen geldig credit card nummer.</target>
```

File: src/translations/validators.nl.xlf [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                <source>This value is not a valid MAC address.</source>
+                <target>Deze waarde is geen geldig MAC address.</target>
+            </trans-unit>
+            <trans-unit id="nif_message">
+                <source>This value is not a valid NIF.</source>
+                <target>Deze waarde is geen geldige NIF.</target>
+            </trans-unit>
+            <trans-unit id="phone_number_message">
+                <source>This value is not a valid phone number.</source>
+                <target>Deze waarde is geen geldig  telefoonnummer.</target>
```
Comment (sstok) [inline src/translations/validators.nl.xlf:51]:
```suggestion
                <target>Deze waarde is geen geldig telefoonnummer.</target>
```

**Review by sstok — CHANGES_REQUESTED** [review]
Some minor typo's. but overall looks good.

**Review by sstok — APPROVED** [review]

**Comment by ikulis** [comment]
@sstok Hi, any change it can be released?

**Comment by sstok** [comment]
I am not a maintainer 😄 🤷🏼‍♂️

**Comment by ikulis** [comment]
@soullivaneuh Hi, any chance this pr can be released?

**Comment by husseinlashin** [comment]
G

**Comment by soullivaneuh** [comment]
@ikulis Merged and released: https://github.com/soullivaneuh/IsoCodesValidator/releases/tag/v4.2.0

Thanks for the contribution and sorry for the big delay!

**Pull request merged**